README.md

```diff
@@ -22,21 +22,41 @@ We use this within our components to publish [statsd](http://github.com/etsy/sta
 
 `IStatsDPublisher` is the interface that you will use in most circumstances. With this you can `Increment` or `Decrement` an event, and send values for a `Gauge` or `Timing`.
 
-The concrete class that implements `IStatsDPublisher` is `StatsDPublisher`. For the constructor parameters, you will need the statsd server host name. You can change the standard port (8125). You can also prepend a prefix to all stats. These values often come from configuration as the host name and/or prefix may vary between test and production environments.
+The concrete class that implements `IStatsDPublisher` is `StatsDPublisher`. The constructor takes an instance of `StatsDConfiguration`. For the configuration's values, you will always need the statsd server host name or IP address. Optionally, you can change the standard port (8125). You can also prepend a prefix to all stats. These values often come from configuration as the host name and/or prefix may vary between test and production environments.
 
 #### Example of setting up a StatsDPublisher
 
-An example of Ioc in NInject for statsd publisher with values from configuration:
+An example of a very simple statsd publisher configuration, using the default values for most things.
+
+```csharp
+
+var statsDConfig = new StatsDConfiguration { Host = "metrics_server.mycompany.com" };
+IStatsDPublisher statsDPublisher = new StatsDPublisher(statsDConfig);
+```
+
+An example of IoC in NInject for statsd publisher with values for all options, read from configuration:
+
 ```csharp
+
+// read config values from app settings
```

Comment:
I think we can address this later. If we do want to talk about config, there's likely more value in talking about the new .Net Core config and IoC bits as they are much less well known.